src/main/java/seedu/address/logic/commands/CommandWords.java

```diff
@@ -152,6 +144,22 @@ public void setCommandWord(String currentWord, String newWord) throws CommandWor
         throw new CommandWordException(getMessageUnused(currentWord));
     }
 
+    /**
+     * throws a (@code CommandWordException) if (@code currentWord) or (@code newWord) is not valid
```

Comment:
Method headers need to be more specific. 
E.g. given a random developer, will they understand what your method is doing (without looking into implementation) by looking at header comments?